app/services/puppet_fact_parser.rb

```diff
@@ -140,6 +140,19 @@ def cores
     facts.dig('processors', 'count') || facts['processorcount']
   end
 
+  def disks_total
+    disks      = facts['disks']
+    disks_size = 0
+
+    if disks.any?
+      disks.values.each do |disk_attrs|
+        disks_size += (disk_attrs['size_bytes'].to_f rescue 0)
+      end
+      # Turning disks_size to closest Mega for easier to read UI
+      (disks_size / 1024 / 1024).ceil if disks_size > 0
```

Comment:
You can use
```suggestion
      disks_size / 1.megabyte
```
for easier readability. 
Also, there is no need to only do it for >0 since 0/1.megabyte is still 0.